src/libs/runtime-utils.js

```diff
@@ -166,3 +167,12 @@ export const appIsSettingUp = app => {
 export const collapsedRuntimeStatus = runtime => {
   return runtime && (runtime.patchInProgress ? 'LeoReconfiguring' : runtime.status) // NOTE: preserves null vs undefined
 }
+
+export const convertedAppStatus = appStatus => {
+  return Utils.switchCase(appStatus,
```

Comment:
```suggestion
  return Utils.switchCase(_.upperCase(appStatus),
```
might be good to ensure the conversion would still work in case `appStatus` is not all uppercase.

Reply:
addressed!